### DISCUSSION TITLE
Trying get help about cs 1.6 eggs, but i gett bann in official discord

### DISCUSSION THREAD

**Discussion by jovimarque** [pterodactyl/panel#5011] [Support] [answered]
This is  a discord support from Pterodactyl official discord  comunity server, i'm just tyring get a counter strike 1.6 egg's, because in  parkervcp gihub there's no egg for counter strike 1.6, and this moderator bann 

![Captura de tela 2024-02-21 082048](https://github.com/pterodactyl/panel/assets/74689585/79cb9336-419f-44e4-a41d-5f92ff9ff397)

**Comment by jovimarque** [comment]
Theres no counter strike 1.6 egg i told to he, but  he just give me a bann, and  no right support, i will migrate 15 thousand  client's for  privade panel, congratulations  moder

![image](https://github.com/pterodactyl/panel/assets/74689585/74003ce3-e41d-46de-a84b-363fa8d0af02)

**Comment by Boy132** [comment, marked as answer]
You were told _multiple_ times that you need the HLDS egg...
